lib/galaxy/datatypes/constructive_solid_geometry.py

```diff
@@ -813,3 +815,63 @@ def get_next_line(fh):
         # Discard the rest of the line
         fh.readline()
     return line.strip()
+
+class Vtu(GenericXml):
+    """Format for defining VTU (VTK Unstructured Grid) data https://docs.vtk.org/"""
+
+    edam_data = "edam:data_3671"
+    edam_format = "edam:format_3621"
+    file_ext = "vtu"
```

Comment:
vtk?

Reply:
?

Reply:
It's called `vtk` files, isn't it?

Reply:
No its called vtu. Its a vtk derivate.

Reply:
I think it would be best to have a data type for any `<VTKFile>` called `vtk` and store the actual type (as given in the type attribute) as metadata (e.g.  `UnstructuredGrid`).